builder/amazon/common/run_config.go

```diff
@@ -84,32 +85,39 @@ func (c *RunConfig) Prepare(ctx *interpolate.Context) []error {
 		c.SSHInterface != "public_dns" &&
 		c.SSHInterface != "private_dns" &&
 		c.SSHInterface != "" {
-		errs = append(errs, errors.New(fmt.Sprintf("Unknown interface type: %s", c.SSHInterface)))
+		errs = append(errs, fmt.Errorf(fmt.Sprintf("Unknown interface type: %s", c.SSHInterface)))
```

Comment:
not a huge deal but the sprint is redundant here

Reply:
@mwhooker Oops - yep! Accidentally left that in. Removed in af7f7f2ce9149e2d7b853e6d91547316c959f92b.